<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_text">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>text_box</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_getconfig">
         <property name="text">
          <string>get config</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox_cameras"/>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="lineEdit_getconfig"/>
       </item>
       <item row="4" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_recive"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_send">
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_search"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_setconfig">
         <property name="text">
          <string>set config</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_recive">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_send"/>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="lineEdit_addcam">
         <property name="text">
          <string>rtsp://192.168.1.95:554/av0_1</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="pushButton_delcam">
         <property name="text">
          <string>del cam</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_search">
         <property name="text">
          <string>search</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_addcam">
         <property name="text">
          <string>add cam</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLineEdit" name="lineEdit_delcam">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLineEdit" name="lineEdit_default_url">
         <property name="text">
          <string>rtsp://192.168.1.95:554/av0_1</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="lineEdit_setconfig"/>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="labe_default_url">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>default url</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_fixedip">
         <property name="text">
          <string>16.208.21.86</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_video">
      <property name="title">
       <string>video_box</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_stop">
         <property name="text">
          <string>stop all</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_video" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_video"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_start">
         <property name="text">
          <string>start all</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
